octoprint/README.md

```diff
@@ -0,0 +1,94 @@
+# Agent Check: OctoPrint
+
+## Overview
+
+This check monitors [OctoPrint][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+To install the OctoPrint check on your host:
+
+
+1. Install the [developer toolkit]
+[3]
+ on any machine.
+
+2. Run `ddev release build octoprint` to build the package.
+
+3. [Download the Datadog Agent][4].
+
+4. Upload the build artifact to any host with an Agent and
+ run `datadog-agent integration install -w
+ path/to/octoprint/dist/datadog_octoprint*.whl`.
+
+### Configuration
+
+1. Edit the `octoprint.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting OctoPrint performance data. See the [sample octoprint.d/conf.yaml][5] for all available configuration options.
+
+2. [Restart the Agent][6].
+
+### Validation
+
+[Run the Agent's status subcommand][7] and look for `octoprint` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][8] for a list of metrics provided by this integration.
+
+### Logs
+
+By default this integration assumes that you are using the [OctoPi][8] image that is pre-configured to run OctoPrint from a Raspberry Pi.
+
+The logs that it collects by default (and their default locations) are:
+
+- OctoPrint App log: `/home/pi/.octoprint/logs`
+- OctoPrint Webcam log: `/var/log/webcamd.log`
+- HA Proxy log: `/var/log/haproxy.log`
+
+Any or all of these may be changed or removed by modifying the integration's `conf.yaml` file.
+
+#### Log Processing
+
+OctoPrint uses it's own log format (not an object format), so making better use of the logs requires creation of a log processing pipeline with some parsing rules.
```

Comment:
```suggestion
OctoPrint uses its own log format (not an object format), so making better use of the logs requires creation of a log processing pipeline with some parsing rules. For example:
```

Reply:
Think this and the next comment are still valid ..